tests/mocks/components.py

```diff
@@ -317,12 +317,11 @@ def __init__(self, name: str):
         super().__init__(name)
 
     async def get_image(self, mime_type: str = CameraMimeType.PNG, **kwargs) -> Union[Image.Image, RawImage]:
-        if not CameraMimeType.is_supported(mime_type) or mime_type == CameraMimeType.RAW:
+        mime_type, is_lazy = CameraMimeType.from_lazy(mime_type)
+        if is_lazy or (not CameraMimeType.is_supported(mime_type)):
             return RawImage(
                 data=self.image.convert("RGBA").tobytes("raw", "RGBA"),
                 mime_type=mime_type,
-                width=self.image.width,
-                height=self.image.height,
             )
         return self.image.copy()
```

Comment:
Does the copy slow us down? Why can't it return the image directly?

Reply:
maybe more a question for @njooma

Reply:
Again, this is just in the tests, so it's moot